devicehive-admin/Dockerfile

```diff
@@ -8,9 +8,9 @@ RUN mkdir -p /opt/devicehive/admin
 ADD admin-start.sh /opt/devicehive/
 
 #installing devicehive admin console
-ADD https://github.com/devicehive/devicehive-admin-console/archive/3.0.0.tar.gz /tmp
-RUN tar -C /tmp -xzf /tmp/3.0.0.tar.gz
-RUN cp -r /tmp/devicehive-admin-console-3.0.0/* /opt/devicehive/admin/
+ADD https://github.com/devicehive/devicehive-admin-console/archive/stable.tar.gz /tmp
+RUN tar -C /tmp -xzf /tmp/stable.tar.gz
+RUN cp -r /tmp/devicehive-admin-console-stable/* /opt/devicehive/admin/
```

Comment:
Lines merged into single RUN instruction.
ADD is not able to extract remote files, only local ones.